docs/quick-start.md

```diff
@@ -544,113 +581,154 @@ For more details, check out the [VMAlert documentation](https://docs.victoriamet
 metadata:
   name: demo
 spec:
   selectAllByDefault: true
-  userNamespaceSelector: {}
-  userSelector: {}
   ingress:
-    class_name: nginx # <-- change this to your ingress-controller
-    host: vm-demo.k8s.orb.local # <-- change this to your domain
-```
-
-**Note** that content of `ingress` field depends on your ingress-controller and domain.
-Your cluster will have them differently. 
-Also, for simplicity, we don't use tls, but in real environments not having tls is unsafe.
+    class_name: 'nginx'                 # <-- Change this to match your Ingress controller (e.g., 'traefik')
+    host: victoriametrics.mycompany.com # <-- Change this to the domain name you’ll use
+EOF
 
-#### VMUser
+kubectl -n vm apply -f vmauth-demo.yaml
 
-To get authorized access to our data it is necessary to create a user using 
-the [vmuser](https://docs.victoriametrics.com/operator/resources/vmuser) resource.
+# Output:
+# vmauth.operator.victoriametrics.com/demo created
+```
+This creates a `VMAuth` named demo and makes it accessible through the domain name `victoriametrics.mycompany.com`. 
+You can use the same method we used before to check if the resources are running.
+Run `kubectl get pod`, but use the label selector `app.kubernetes.io/name=vmauth`.
 
-Create file `vmuser.yaml` 
+Check if the `Ingress` rule was created:
+```sh
+kubectl get ingress -n vm
 
-```shell
-code vmuser.yaml
+# Output:
+# NAME          CLASS   HOSTS                           ADDRESS   PORTS   AGE
+# vmauth-demo   nginx   victoriametrics.mycompany.com             80      5m
 ```
 
-with the following content:
+Next, we’ll define a user that can securely access storage VMUI and alerting UI, as well as run queries we used in previous chapters. 
+You can also expand access later by adding more paths you needed.
 
-```yaml
+Create a new file named `vmuser-demo.yaml` with the following content:
+```sh
+cat <<'EOF' > vmuser-demo.yaml
 apiVersion: operator.victoriametrics.com/v1beta1
 kind: VMUser
 metadata:
   name: demo
 spec:
-    name: demo
-    username: demo
-    generatePassword: true
-    targetRefs:
-      # vmui + vmselect
-      - crd:
-          kind: VMCluster/vmselect
-          name: demo
-          namespace: vm
-        target_path_suffix: "/select/0"
-        paths:
-          - "/vmui"
-          - "/vmui/.*"
-          - "/prometheus/api/v1/query"
-          - "/prometheus/api/v1/query_range"
-          - "/prometheus/api/v1/series"
-          - "/prometheus/api/v1/status/.*"
-          - "/prometheus/api/v1/label/"
-          - "/prometheus/api/v1/label/[^/]+/values"
-```
-
-After that you can deploy `vmauth` and `vmuser` resources to the kubernetes cluster:
-
-```shell
-kubectl apply -f vmauth.yaml -n vm
-kubectl apply -f vmuser.yaml -n vm
+  name: demo
+  username: demo
+  generatePassword: true
+  targetRefs:
+    # vmsingle
+    - crd:
+        kind: VMSingle
+        name: demo
+        namespace: vm
+      paths:
+        - "/vmui.*"
+        - "/api/v1/query"
```

Comment:
should we use `/api/v1/.*` so all read pathes are available?
otherwise, things like Graph, Raw Query, Cardinality Explorer won't work in vmui

Reply:
Is there any /api/v1/xxx api that should not be exposed ?

Reply:
For demo? I think no.
For production, administrators may need to protect /api/v1/admin handlers and follow other instructions at https://docs.victoriametrics.com/#security

Reply:
Updated. Added the only API required by UI (tested)